libs/designsystem/src/lib/components/tabs/tab-button/tab-button.component.scss

```diff
@@ -37,6 +37,15 @@ ion-tab-button {
     .icon-container {
       margin-right: utils.size('xxs');
     }
+
+    @include utils.slotted(div kirby-badge) {
+      &.md {
+        // Magic constants here as we have no tokens that gives
+        // badge the correct position in relation to the tab icon
+        --kirby-badge-top: -10px;
+        --kirby-badge-left: 14px;
```

Comment:
Since we're already using magic numbers I suggest we try to use relative units, i.e. `em`. That should make the positioning more robust. `-10px` and `14px` only works for one specific font size.

Once we solve https://github.com/kirbydesign/designsystem/issues/2313 we may be able to use size tokens for this.